src/test/resources/nil.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please add the [license header](https://github.com/OfficeDev/ews-java-api/blob/master/license.txt) to all new files.